playground/recursive-set.nix

```diff
@@ -0,0 +1,6 @@
+rec {
+    foo.bar = 13;
+    xyz = foo;
```

Comment:
Perhaps it's just my editor, but when hovering over `foo` here, I don't see an attr-set.

Reply:
Shows `{ bar }` now. seems expected.